python/hsfs/constructor/serving_prepared_statement.py

```diff
@@ -0,0 +1,100 @@
+#  Copyright 2021. Logical Clocks AB
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+import humps
+import json
+
+from hsfs import util
+from hsfs.constructor import prepared_statement_parameter
+
+
+class ServingPreparedStatement:
+    def __init__(
+        self,
+        feature_group_id=None,
+        prepared_statement_index=None,
+        prepared_statement_parameters=None,
+        query_online=None,
+        type=None,
+        items=None,
+        count=None,
+        href=None,
+    ):
+
+        self._feature_group_id = feature_group_id
+        self._prepared_statement_index = prepared_statement_index
+        self._prepared_statement_parameters = [
+            prepared_statement_parameter.PreparedStatementParameter.from_response_json(
+                psp
+            )
+            for psp in prepared_statement_parameters
+        ]
+        self._query_online = query_online
+
+    @classmethod
+    def from_response_json(cls, json_dict):
+        json_decamelized = humps.decamelize(json_dict)
+        if json_decamelized["count"] >= 1:
+            return [cls(**pstm_dto) for pstm_dto in json_decamelized["items"]]
+        return cls(**json_decamelized)
+
+    def update_from_response_json(self, json_dict):
+        json_decamelized = humps.decamelize(json_dict)
+        _ = json_decamelized.pop("type")
+        _ = json_decamelized.pop("href")
```

Comment:
same as before, since you have the fields, no need to drop it as long as it's not a property accessible by the user